contracts/MechMarketplace.sol

```diff
@@ -803,8 +803,8 @@ contract MechMarketplace is IErrorsMarketplace {
         address account,
         bytes memory data,
         uint256 nonce
-    ) public view returns (uint256 requestId) {
-        requestId = uint256(keccak256(
```

Comment:
`bytes32` is basic for signatures, so conceptually better to switch to that one. Got green light from launch team.